app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'benefits_intake_service/service'
+
+class BenefitsIntakeStatusJob
+  include Sidekiq::Job
+
+  def perform
+    pending_form_submission_ids = FormSubmission
+                                  .joins(:form_submission_attempts)
+                                  .where(form_submission_attempts: { aasm_state: 'pending' })
+                                  .map(&:benefits_intake_uuid)
+    response = BenefitsIntakeService::Service.new.get_bulk_status_of_uploads(pending_form_submission_ids)
+    handle_response(response)
+  end
+
+  private
+
+  def handle_response(response)
+    response.body['data'].each do |submission|
+      if submission.dig('attributes', 'status') == 'error' || submission.dig('attributes', 'status') == 'expired'
+        handle_failure(submission)
+      elsif submission.dig('attributes', 'status') == 'vbms'
+        handle_success(submission)
+      end
+    end
+  end
+
+  def handle_failure(submission)
+    form_submission = FormSubmission.find_by(benefits_intake_uuid: submission['id'])
+    form_submission_attempt = form_submission
+                              .form_submission_attempts
+                              .where(aasm_state: 'pending')
+                              .order(created_at: :asc)
+                              .last
+    form_submission_attempt.fail!
+  end
+
+  def handle_success(submission)
+    form_submission = FormSubmission.find_by(benefits_intake_uuid: submission['id'])
+    form_submission_attempt = form_submission
+                              .form_submission_attempts
+                              .where(aasm_state: 'pending')
+                              .order(created_at: :asc)
+                              .last
+    form_submission_attempt.succeed!
```

Comment:
please add a datadog info so we can build a dashboard. From the ticket: 

> For each submission that changes to a Success state
Log an info message that indicates the successful submission, with FormSubmission ID and FormSubmission Form Type as part of logging payload

Reply:
@tblackwe for this and the error state we have [a log in the `FormSubmissionAttempt`](https://github.com/department-of-veterans-affairs/vets-api/blob/3d1ce283b94327564897189830d1ef8a943052e7/app/models/form_submission_attempt.rb#L33). Is that sufficient? I feel like someone told me that `Rails.logger` goes to DataDog too, but I can't seem to find it now. Maybe it's better to remove it from there and put the logs here?

Reply:
that should be sufficient, thanks! forgot it was there